src/opnsense/mvc/app/views/OPNsense/Unbound/dnsbl.volt

```diff
@@ -27,10 +27,21 @@
 
 <script>
    $(document).ready(function() {
+       function nxdomain_toggle() {
+           $('input[id="unbound.dnsbl.nxdomain"]').is(':checked') ?
+               $('input[id="unbound.dnsbl.address"]').attr('disabled', 'disabled') :
+               $('input[id="unbound.dnsbl.address"]').removeAttr('disabled');
+       }
+
        var data_get_map = {'frm_dnsbl_settings':"/api/unbound/settings/get"};
        mapDataToFormUI(data_get_map).done(function(data){
            formatTokenizersUI();
            $('.selectpicker').selectpicker('refresh');
+
+           $('input[id="unbound.dnsbl.nxdomain"]').click(function() {
```

Comment:
it's easier to bind to `change()`, like :

https://github.com/opnsense/core/blob/b6f95cdea429d243f1e70f55ce817407264eb60f/src/opnsense/mvc/app/views/OPNsense/Firewall/alias.volt#L227

Reply:
Removed it as per your next comment for simplicity's sake.